misc-tools/dj_make_chroot.in

```diff
@@ -384,17 +384,17 @@ chroot "$CHROOTDIR" /bin/sh -c "apt-get clean"
 
 # Remove unnecessary setuid bits
 chroot "$CHROOTDIR" /bin/sh -c "chmod a-s /usr/bin/wall /usr/bin/newgrp \
-	/usr/bin/chage /usr/bin/chfn /usr/bin/chsh /usr/bin/expiry \
-	/usr/bin/gpasswd /usr/bin/passwd \
-	/bin/su /bin/mount /bin/umount /sbin/unix_chkpwd" || true
+  /usr/bin/chage /usr/bin/chfn /usr/bin/chsh /usr/bin/expiry \
+  /usr/bin/gpasswd /usr/bin/passwd \
+  /bin/su /bin/mount /bin/umount /sbin/unix_chkpwd" || true
 
 # Disable root account
 sed -i "s/^root::/root:*:/" "$CHROOTDIR/etc/shadow"
 
 # Create a file to test that in the judging environment no
 # access to group root readable files is available:
 echo "This file should not be readable inside the judging environment!" \
-	> "$CHROOTDIR/etc/root-permission-test.txt"
+  > "$CHROOTDIR/etc/root-permission-test.txt"
```

Comment:
This indentation looks off.

Reply:
The rationale was to indent using 4 spaces, but use 2 for multi-line commands. If you prefer I'll use 4 spaces everywhere.